.eslintrc.json

```diff
@@ -85,7 +93,8 @@
         "no-var": "error",
         "eqeqeq": "error",
         "object-curly-spacing": ["error", "always"],
-        "object-shorthand": "error"
+        "object-shorthand": "error",
+        "react/prop-types": 0
```

Comment:
We just disable warning for props used without prop-types declaration, because we don't use it yet. https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md